schema/schema.sql

```diff
@@ -0,0 +1,26 @@
+--Create database
+CREATE DATABASE my_catalog;
+
+-- Create item table
+CREATE TABLE item (
+  id INT PRIMARY KEY,
+  genre_id INT REFERENCES genre(id),
+  publish_date DATE,
+  archived BOOLEAN,
+);
+
```

Comment:
We may not need this since the instruction is music_albums table (add all properties and associations from the parent Item class as table columns). I think this will be duplication of data